tests/ReflectionClassConstantTest.php

```diff
@@ -0,0 +1,108 @@
+<?php
+declare(strict_types=1);
+
+namespace Go\ParserReflection;
+
+use Go\ParserReflection\Stub\Foo;
+use Go\ParserReflection\Stub\SubFoo;
+use TestParametersForRootNsClass;
+use Go\ParserReflection\Stub\ClassWithPhp71Features;
+
+/**
+ * @requires PHP 7.1
```

Comment:
Please, bump `composer.json` requirement to be greater than `7.1`, I will publish a new `2.1.0` version

Reply:
And badge content in the README for PHP > 7.1 )

Reply:
Done! Note: I removed the polyfill.php completly because it isn't needed anymore with a requirement of php 7.1.